manifests/common/connection-types/oci-v1.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: oci-v1
+  labels:
+    opendatahub.io/connection-type: 'true'
+    opendatahub.io/dashboard: 'true'
+  annotations:
+    openshift.io/description: 'Connect to an OCI-compliant container registry, enabling integration with containerized applications and services. Use this connection type to pull and manage container images and artifacts that adhere to the Open Container Initiative (OCI) standards, ensuring compatibility with OCI-compliant tools and workflows.'
+    openshift.io/display-name: OCI compliant registry - v1
+data:
+  category: '["Container registry"]'
+  fields: '[{"type":"dropdown","name":"Access type","description":"Access type can be push (push for uploading data), pull (pull for accessing resources), or both depending on the use case. E.g. Pull/both for deploying a model, push/both for InstructLab fine-tune runs.","envVar":"ACCESS_TYPE","required":false,"properties":{"variant":"multi","items":[{"label":"Push secret","value":"Push"},{"label":"Pull secret","value":"Pull"}]}},{"type":"file","name":"Secret details","envVar":".dockerconfigjson","required":true,"properties":{"extensions":[".dockerconfigjson",".json"]}},{"type":"short-text","name":"Base URL / Registry URI","envVar":"OCI_HOST","required":true,"properties":{}}]'
```

Comment:
There's ongoing discussion about whether "InstructLab" branding is ok to use in upstream ODH or whether we need something generic like "LAB tune". @Gkrumbach07 may have newer info on that but I think [this doc](https://docs.google.com/document/d/1KMrAfGOS6Nz-bgHdOrbzDmac6VWlgv8NAV0h3mVcwq8/edit?tab=t.0#heading=h.j5cfi9daol6m) is where that decision is happening. We probably shouldn't block the PR on this but we may want to loop back and make sure this is changed if necessary before release